File: backend/src/main/scala/com/skewwhiffy/auraltester/controller/InfoController.scala
package com.skewwhiffy.auraltester.controller

import com.skewwhiffy.auraltester.model.ApplicationInformation
import org.springframework.web.bind.annotation.{GetMapping, RequestMapping, RestController}

@RestController
@RequestMapping(path = Array("/info"))
class InfoController {
  @GetMapping
  def get():ApplicationInformation = new ApplicationInformation("0.0.1")
}

File: backend/src/main/java/com/skewwhiffy/auraltester/controller/InfoController.java
package com.skewwhiffy.auraltester.controller;

import com.skewwhiffy.auraltester.dto.InfoResponse;
import com.skewwhiffy.auraltester.service.InfoService;
import lombok.AllArgsConstructor;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@AllArgsConstructor
public class InfoController {
    private final InfoService infoService;

    @GetMapping("/api/info")
    public InfoResponse get() {
        return infoService.get();
    }
}
